gem_city.rb

```diff
@@ -0,0 +1,68 @@
+class TypeError < StandardError; end
+
+# start GemCity class
+class GemCity
+  attr_reader :thieves, :officers, :population, :civilians
+  # This class represents the town of GemCity
+  # This is a town riddled with crime but we can find out how happy the town is
+
+  def initialize
+    @thieves = 5
+    @officers = 1
+    @population = 50
+    @civilians = @population - @thieves - @officers
+  end
+
+  def thieves=(number)
+    fail TypeError, 'Input must be an integer' unless number.is_a? Integer
```

Comment:
Check out the `Integer` method: http://ruby-doc.org/core-2.0.0/Kernel.html#method-i-Integer

```ruby
Integer(42) #=> 42
Integer(:foo)
#=> TypeError: can't convert Symbol into Integer
```